src/Tizen.Sensor/Tizen.Sensor/Sensor.cs

```diff
@@ -545,5 +604,600 @@ private void DestroyHandles()
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error in setting range of sensor provider");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.ProviderSetRange Failed");
+            }
+
+        }
+
+        /// <summary>
+        /// Sets the resolution of sensor values to the sensor provider.
+        /// </summary>
+        /// <param name="resolution">
+        /// The resolution.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void ProviderSetResolution(float resolution)
+        {
+            int error = Interop.SensorProvider.ProviderSetResolution(_sensorProviderHandle, resolution);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error in setting resolution of sensor provider");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.ProviderSetResolution Failed");
+            }
+
+        }
+
+        /// <summary>
+        /// Registers the callback function to be invoked when a listener starts the sensor provider.
+        /// </summary>
+        /// <param name="userData">
+        /// The user data to be passed to the callback function
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void ProviderSetStartCb(IntPtr userData)
+        {
+            _sensorProviderStartCallBack = (IntPtr _provider, Int64 _userData) =>
+            {
+                ProviderStartEventArgs e = new ProviderStartEventArgs(_userData)
+                {
+                };
+                ProviderStarted?.Invoke(null, e);  
+            };
+
+            int error = Interop.SensorProvider.ProviderSetStartCb(_sensorProviderHandle, _sensorProviderStartCallBack, userData);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error in setting start callback of sensor provider");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.ProviderSetStartCb Failed");
+            }
+
+        }
+
+        private void ProviderSetStopCb(IntPtr userData)
+        {
+            _sensorProviderStopCallBack = (IntPtr _provider, Int64 _userData) =>
+            {
+                ProviderStopEventArgs e = new ProviderStopEventArgs(_userData)
+                {
+                };
+                ProviderStoped?.Invoke(null, e);
+            };
+
+            int error = Interop.SensorProvider.ProviderSetStopCb(_sensorProviderHandle, _sensorProviderStopCallBack, userData);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error in setting stop callback of sensor provider");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.ProviderSetStopCb Failed");
+            }
+
+        }
+
+        private void ProviderSetIntervalChangedCb(IntPtr userData)
```

Comment:
Who is use it?
I can't found. 
And it is private

Reply:
ProviderSetIntervalChangedCb access level changed to public